build-logic/gradle.properties

```diff
@@ -0,0 +1,5 @@
+org.gradle.unsafe.configuration-cache=true
+# TODO last two errors here: https://youtrack.jetbrains.com/issue/KT-43605
+#  should be ok to use it because properties are `java.rmi.server.hostname` and `kotlin.environment.keepalive`
+#  which should not change on local machine
+org.gradle.unsafe.configuration-cache-problems=fail
```

Comment:
should be `warn` here, but somehow it works for me;
Please try it, checking out the branch and running any task